website/static/js/registrationUtils.js

```diff
@@ -260,10 +260,15 @@ var MetaSchema = function(params) {
     self.fulfills = params.fulfills || [];
     self.messages = params.messages || {};
 
+    var count = 0;
     $.each(self.schema.pages, function(i, page) {
         var mapped = {};
         $.each(page.questions, function(qid, question) {
-            mapped[qid] = new Question(question, qid);
+            // This is necessary now since page.questions is a list
```

Comment:
Talk to Lyndsy per schema structure documentation- he may have some notes to help move that discussion along. Although we can fake schemas by example, knowing what data to include would help us push out new registration types in the future.

Right now we have no guarantees our schemas are complete, which creates the potential for edge case bugs down the line. Functions like `ensure_schema_structure` imply that our schemas need to be corrected for omissions at runtime, which is kind of freaky.